Source/WebCore/svg/SVGAElement.cpp

```diff
@@ -105,7 +105,7 @@ void SVGAElement::svgAttributeChanged(const QualifiedName& attrName)
 
 RenderPtr<RenderElement> SVGAElement::createElementRenderer(RenderStyle&& style, const RenderTreePosition&)
 {
-    if (is<SVGElement>(parentNode()) && downcast<SVGElement>(*parentNode()).isTextContent())
+    if (auto* svgParent = dynamicDowncast<SVGElement>(parentNode()); svgParent && svgParent->isTextContent())
```

Comment:
This is hard to read. Can you please move the casting out of the if-statement.

Reply:
Then the scope of the variable is way larger than needed, plus it's a raw pointer. I'll make the change since you're maintaining SVG but I think my code was better.